org.freeplane.App.yml

```diff
@@ -0,0 +1,69 @@
+app-id: org.freeplane.App
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: freeplane
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=home
+
+  - --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+  - --env=JAVA=/app/jre/bin/java
+
+build-options:
+    env:
+        JAVA_HOME: /usr/lib/sdk/openjdk11/
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+
+  - name: freeplane
+    no-autogen: true
+    sources:
+      # fetch source code for getting some resources
+      - type: git
+        url: https://github.com/freeplane/freeplane.git
+        tag: release-1.8.2
+
+      # fetch binary for packing without compiling
+      - type: archive
+        url: https://sourceforge.net/projects/freeplane/files/freeplane%20stable/freeplane_bin-1.8.2.zip
+        sha256: 149916701bc8aabff1ac1e30c5296e51b28279cb26448a219ec0228620657245
+        strip-components: 0
+      
+      # Install AppData.
+      - type: file
+        path: org.freeplane.App.metainfo.xml
+
+      - type: patch
+        path: freeplane.desktop.patch
+        strip-components: 0
+
+    buildsystem: simple
+    build-commands: 
+      - mkdir -p /app/Freeplane /app/bin
+      - cp -ra freeplane-1.8.2/* /app/Freeplane
```

Comment:
Nitpick: you probably want to use a regex here instead of a version just to avoid the burden of changing it in multiple places.

Reply:
You mean regex or wildcards?

Anyhow, the version number is used two times: here and at the git branch tag, so manifest needs to be updated by hand in any case.